### PR TITLE
(DOCSP-7107): New connection ux

### PR DESCRIPTION
**Pull request by jeff-allen-mongo**
To verify this new ux, you'll need to download the 1.20 beta version of Compass from the download center. I'd like feedback on the overall structure of this page and if you think it can be improved. Some things  considered but ultimately did not do:

- Tab out based on deployment. E.g., have one tab set for connecting to a standalone, one for a replica set / sharded cluster

- Tab out based on whether you are connecting to an Atlas deployment or a self-hosted deployment. Instead of doing this, I called out Atlas-specific instructions where appropriate.

This is largely a refactor of existing content.

Staged: https://docs-mongodbcom-staging.corp.mongodb.com/compass/docsworker/DOCSP-7107/connect.html

### CODE REVIEW

File: source/includes/steps-starting-compass.yaml [hunk 1]
```diff
@@ -5,64 +5,459 @@ content: |
...
+               passphrase used to decrypt your private key (stored
+               in the specified identity file). A passphrase
+               provides an extra layer of security for an SSH
+               connection.
+
+           * - :guilabel:`SSH Password`
+             -  The password used to secure the SSH connection.
+                This is required if you are not using an identity file.
+
+        To use an SSH tunnel through a bastion host, the ssh
```
Comment (steveren) [inline source/includes/steps-starting-compass.yaml]: Capitalize "ssh" on lines 425 and 430

File: source/connect.txt [hunk 2]
```diff
@@ -12,12 +12,26 @@ Connect to MongoDB
...
+  :manual:`primary </core/replica-set-primary/>` member to a
+  :manual:`secondary </core/replica-set-secondary/>` or vice versa as
+  the result of an election, |compass-short| may either forcibly close
+  the connection or display stale data.
+
+  Instead, to connect to a replica set, either the replica set
+  :guilabel:`SRV record` or :guilabel:`Replica Set Name` when 
+  filling in your connection information.
+
+- .. include:: /includes/fact-non-genuine-warning.rst
```
[the remark this answers is not in the document]
Comment (jeff-allen-mongo) [reply source/connect.txt]: Yes or CosmosDB or the like. I believe it was added in Compass 1.18.

File: source/includes/steps-starting-compass.yaml (hunk 1 above)
Comment (jeff-allen-mongo) [reply source/includes/steps-starting-compass.yaml]: Good catch. Done.

File: source/connect.txt [hunk 3]
```diff
@@ -12,12 +12,26 @@ Connect to MongoDB
...
+
+- When connecting |compass-short| to a replica set, it is not
+  recommended to connect directly to an individual replica
+  set member. If the member to which you are connected switches from a
+  :manual:`primary </core/replica-set-primary/>` member to a
+  :manual:`secondary </core/replica-set-secondary/>` or vice versa as
+  the result of an election, |compass-short| may either forcibly close
+  the connection or display stale data.
+
+  Instead, to connect to a replica set, either the replica set
```
[the remark this answers is not in the document]
Comment (jeff-allen-mongo) [reply source/connect.txt]: Good catch. Done.

**Comment by jeff-allen-mongo** [comment]
@steveren this is ready for another look. I did some shuffling around of how this is presented so now we have two separate step files, one for the paste your string and one for the individual form fields. I removed the "Open Compass" step because it seemed unnecessary and also removing it made it easier to structure the content in this new way. Lmk what you think!

**Comment by jeff-allen-mongo** [comment]
@mmarcon, would you mind reviewing these updates for the new Connection ux?

**Pull request merged**